RealmeParts: Enable 90 Hz during game mode

aniketghodake10 opened the pull request:
* Will switch to 60 Hz when game mode is disabled
* It is for 60 Hz users who want 90 Hz only while gaming
* If your toggle is 90 Hz or Smooth display, then Game FPS will do 
nothing as it will consider you as full time 90 Hz user

4j17h reviewed (commented):
> If your toggle is 90 Hz or Smooth display, then Game FPS will do 
nothing as it will consider you as full time 90 Hz user

Well then, disable the Game FPS toggle if smooth display is enabled or 90 hz is already set

aniketghodake10 commented:
> > If your toggle is 90 Hz or Smooth display, then Game FPS will do
> > nothing as it will consider you as full time 90 Hz user
> 
> Well then, disable the Game FPS toggle if smooth display is enabled or 90 hz is already set

Yes

karthick111 commented:
Clone app source again & force push in your fork to update this PR

aniketghodake10 commented:
@4j17h @karthick111 Pushed, plz check.

aniketghodake10 committed:
RealmeParts: Enable 90 Hz during game mode

*Will switch to 60 Hz when game mode is disabled
*It is for 60 Hz users who want 90 Hz only while gaming
*If your toggle is 90 Hz or Smooth display, then Game FPS will do
nothing as it will consider you as full time 90 Hz user

Other changes:
*Fixed Glitched RefreshTile
*Now prefrences are changed upon tile click (refresh rate only, for game 
mode dont know whether we can make mGameModeSwitch as public)
*Disabled Smooth display on A12
* App is now robust(added few checks in DeviceSettings.java, checks when 
app installed/updated)

aniketghodake10 committed:
Fixed Tile Change Lag

*Before:game mode and refresh TileChange lags or crashes when realme 
parts is not open in background
*Reason: If used RefreshRateSwitch and DeviceSettings methods in OnCLick 
method
